app/src/services/wiseoldman.ts

```diff
@@ -145,6 +146,12 @@ export const getPlayerCompetitions = cache((username: string) => {
   return handleNotFound(apiClient.players.getPlayerCompetitions(username));
 });
 
+export const getPlayerCompetitionStandings = cache(
+  (username: string, filter: PlayerCompetitionsFilter) => {
```

Comment:
due to how React's cache works, parameters must be primitives.

So it's best pass in "status" instead

Reply:
done